assembly/src/assembler/instruction/ext2_ops.rs

```diff
@@ -115,14 +115,14 @@ pub fn ext2_inv(span: &mut SpanBuilder) -> Result<Option<CodeBlock>, AssemblyErr
     span.push_advice_injector(Ext2Inv);
     #[rustfmt::skip]
     let ops = [
-        AdvPop,   // [a0', a1, a0, ...]
-        AdvPop,   // [a1', a0', a1, a0, ...]
-        Ext2Mul,  // [a1', a0', 0, 1, ...]
-        MovUp2,   // [0, a1', a0', 1, ...]
-        Eqz,      // [1, a1', a0', 1, ...]
-        Assert,   // [a1', a0', 1, ...]
-        MovUp2,   // [1, a1', a0', ...]
-        Assert    // [a1', a0', ...]
+        AdvPop,         // [a0', a1, a0, ...]
+        AdvPop,         // [a1', a0', a1, a0, ...]
+        Ext2Mul,        // [a1', a0', 0, 1, ...]
+        MovUp2,         // [0, a1', a0', 1, ...]
+        Eqz,            // [1, a1', a0', 1, ...]
+        Assert(ZERO),   // [a1', a0', 1, ...]
```

Comment:
I think it could be interesting to give unique codes for some of these errors. This way, if an error happens, a user could figure out that this was because something went wrong in `ext2_div` instruction.

However, one thing we should do first it come up with some scheme for error codes so that instruction errors don't overlap with `stdlib` errors, and these don't overlap with custom library errors etc.